src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptionPages.java

```diff
@@ -245,7 +245,7 @@ public static OptionPage performance() {
                 .add(OptionImpl.createBuilder(int.class, sodiumOpts)
                         .setName(Text.translatable("sodium.options.chunk_update_threads.name"))
                         .setTooltip(Text.translatable("sodium.options.chunk_update_threads.tooltip"))
-                        .setControl(o -> new SliderControl(o, 0, Runtime.getRuntime().availableProcessors(), 1, ControlValueFormatter.quantityOrDisabled("threads", "Default")))
+                        .setControl(o -> new SliderControl(o, 0, Runtime.getRuntime().availableProcessors(), 1, ControlValueFormatter.quantityOrDisabled("sodium.options.chunk_update_threads.value", "options.gamma.default")))
```

Comment:
I did it a bit differently in #2283, but I could use your variant if you'd like to PR there

Reply:
That's no problem.